src/openttd.cpp

```diff
@@ -1009,6 +1012,28 @@ bool SafeLoad(const std::string &filename, SaveLoadOperation fop, DetailedFileTy
 	return false;
 }
 
+static void UpdateSocialIntegration(GameMode game_mode)
+{
+	switch (game_mode) {
+		case GM_BOOTSTRAP:
+		case GM_MENU:
+			SocialIntegration::EventEnterMainMenu();
+			break;
+
+		case GM_NORMAL:
+			if (_network_server) {
```

Comment:
```suggestion
			if (_networking) {
```
Or is playing as a client considered single player?